spec/01-unit/006-schema_validation_spec.lua

```diff
@@ -64,6 +64,45 @@ describe("Schemas", function()
       end)
     end)
 
+    describe("[string]", function()
+      it("should trim whitespace from value with no trim_whitespace property set", function()
+        local values = {string = " kong "}
+
+        local valid, err = validate_entity(values, schema)
+        assert.True(valid)
+        assert.falsy(err)
+        assert.are.same("kong", values.string)
+      end)
+      it("should trim whitespace from value with trim_whitespace = true", function()
+        local trim_schema = {
+          fields = {
+            string = { type = "string", trim_whitespace = true},
+          }
+        }
+
+        local values = {string = " kong "}
+
+        local valid, err = validate_entity(values, trimSchema)
```

Comment:
Your patch is now failing to lint (https://travis-ci.org/Kong/kong/jobs/409120621), because the name was not updated here and in the below test